addons/medical_damage/stringtable.xml

```diff
@@ -2,16 +2,16 @@
 <Project name="ACE">
     <Package name="Medical_Damage">
         <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_DisplayName">
-            <English>Player Damage Threshold</English>
+            <English>Player Critical Damage Threshold</English>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_Description">
-            <English>Sets the amount of damage a player can take before being killed.</English>
+            <English>Sets the amount of damage a player can receive before going unconscious.</English>
```

Comment:
🤔 I still think the description could be better (because it sounds cumulative here, but it's really for a single damage event). However, we can tweak it later 👍